docs/jvminterop-goals.pod

```diff
@@ -1,10 +1,10 @@
-=encoding utf8
+Raku=encoding utf8
```

Comment:
This feels wrong?

Reply:
Fixed the typo now. Thank you for spotting.